src/AXP192.cpp

```diff
@@ -527,31 +527,32 @@ void AXP192::SetLCDRSet(bool state)
     Write1Byte(reg_addr, data);
 }
 
+// Select source for BUS_5V
+// 0 : use internal boost
+// 1 : powered externally
 void AXP192::SetBusPowerMode(uint8_t state)
 {
     uint8_t data;
     if (state == 0)
     {
+        // Set GPIO to 3.3V (LDO OUTPUT mode)
         data = Read8bit(0x91);
-        Write1Byte(0x91, (data & 0X0F) | 0XF0);
-
+        Write1Byte(0x91, (data & 0x0F) | 0xF0);
+        // Set GPIO0 to LDO OUTPUT, pullup N_VBUSEN to disable VBUS supply from BUS_5V
         data = Read8bit(0x90);
-        Write1Byte(0x90, (data & 0XF8) | 0X02); //set GPIO0 to LDO OUTPUT , pullup N_VBUSEN to disable supply from BUS_5V
-
-        data = Read8bit(0x91);
-
-        data = Read8bit(0x12);         //read reg 0x12
-        Write1Byte(0x12, data | 0x40); //set EXTEN to enable 5v boost
+        Write1Byte(0x90, (data & 0xF8) | 0x02);
+        // Set EXTEN to enable 5v boost
+        data = Read8bit(0x10);
```

Comment:
@felmue : Are you sure that reg 0x10 is the right one? 0x12 also hast EXTEN enable and disable control. How did you find out that this is the right one?

Reply:
Hello @trunneml 

I appreciate you looking into this issue as well. I was consulting this [datasheet](https://github.com/m5stack/M5-Schematic/blob/master/Core/AXP192%20Datasheet_v1.1_en_draft_2211.pdf) at the time and in that one only register 0x10 controls EXTEN. Which makes me wonder about maybe some stuff got lost / moved around when it was translated from CN to English?

Thanks
Felix

Reply:
Hi @felmue ,

there is a note in the chines data sheet below REG12H. Using Google translator: 

> Note: REG12Hbit6/4 correspond to REG10Hbit2/0 respectively. 

So I think using register 12 or register 10 ends in the same result and there is no need to change part of the code. 
Beside that your PR looks good. :)

Bye
Michael